backend/src/v5/models/teamspaces.js

```diff
@@ -118,9 +112,16 @@ Teamspace.updateAddOns = async (teamspace, addOns) => {
 
 Teamspace.removeAddOns = (teamspace) => teamspaceUpdate({ user: teamspace }, { $unset: possibleAddOns });
 
-Teamspace.getTeamspaceAdmins = async (ts) => {
-	const data = await getTeamspace(ts, { 'customData.permissions': 1 });
-	return data.customData.permissions.flatMap(
+Teamspace.getTeamspaceAdmins = async (teamspace) => {
+	const tsSettings = await db.findOne(
```

Comment:
At this point getTeamspace should be changed to fetching teamspace settings so you should still be using that function.

If this isn't the case then whatever still needing data from user bson should be migrated (I think)